src/clients/meta/MetaClient.h

```diff
@@ -599,6 +599,10 @@ class MetaClient {
 
   folly::Future<StatusOr<bool>> ingest(GraphSpaceID spaceId);
 
+  HostAddr getMetaLeader() { return leader_; }
+
+  DateTime getLastHeartBeatTime() { return lastHeartBeatTime_; }
```

Comment:
rename to `HeatbeatTime`

Reply:
I mean heartbeat is one word not two words, please don't capitalize the part `beat`. Use `Heartbeat`